### PR TITLE
Tests! And travis prep

### PR DESCRIPTION
**Pull request by aladage**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,38 +1,50 @@
...
-    "chalk": "^2.4.2",
-    "node-typewriter": "^0.0.2",
-    "inquirer": "^6.2.2",
-    "fs-extra": "^7.0.1",
-    "simple-git": "^1.107.0",
-    "replace-in-file": "^3.4.3",
-    "child-process-promise": "^2.2.1",
-    "semver-regex": "^2.0.0"
-  }
+    "name": "orko",
```
Comment (scheul93) [inline package.json]: to be consistent with other Skeletor repos, consider renaming to `@deg-skeletor/orko`

File: travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+- node
+cache:
+  directories:
+  - node_modules
+deploy:
+  provider: npm
+  email: scheul93@gmail.com
```
Comment (scheul93) [inline travis.yml]: Did you generate your own Travis key?
I think it uses your email as part of the encryption

File: package.json [hunk 3]
```diff
@@ -1,38 +1,50 @@
...
+    "version": "1.0.0",
```
Comment (heaper) [inline package.json:3]: What an optimistic version number you've chosen!

Comment (aladage) [reply package.json:3]: Thank you!

File: tests/onboarding.test.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+const {hello} = require('../config/text.js');
+const onboarding = require('../steps/01_onboarding.js');
+
+test('Onboarding says hello', async () => {
```
Comment (heaper) [inline tests/onboarding.test.js:4]: heart eyes emoji!

**Pull request merged**